mlrun/runtimes/sparkjob.py

```diff
@@ -232,7 +232,12 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
                             meta.name, driver, status
                         )
                     )
-                    logger.info("use .watch({}) to see logs".format(meta.name))
+                    resp = self.get_job(meta.name, meta.namespace)
+                    ui_ingress = get_in(resp, "status.driverInfo.webUIIngressAddress")
```

Comment:
I don't really like this `get_in` thing, didn't do a refactor to remove it, but prefer not adding new usages of it, I prefer the classic way of doing it:
`resp.get('status', {}).get('driverInfo', {}).get('webUIIngressAddress')` (assuming all middle values are indeed support to be dicts)

Reply:
not that I understand why, but done